package.json

```diff
@@ -42,23 +43,36 @@
     "build-fonts": "gulp fonts",
     "build-img": "gulp images",
     "build-js": "gulp javascript",
+    "build-uswds": "gulp build-uswds-if-needed",
     "check-contrast": "node ./utils/check-contrast.js",
     "clean": "gulp clean-assets",
     "crawl": "node config/crawl.js",
     "federalist": "npx gulp build",
     "lint": "gulp eslint scss-lint",
     "prestart": "gulp build",
-    "serve": "bundle exec jekyll serve --drafts --future --incremental --livereload",
+    "proof": "bundle exec htmlproofer --assume-extension --disable-external --allow-hash-href --alt-ignore '/.*/' --file-ignore '/developers/' ./_site",
+    "serve": "npm run build-all-assets && bundle exec jekyll serve --drafts --future --incremental --livereload",
+    "serve:package": "npm run build-uswds && npm run serve",
+    "serve:local": "export FRACTAL_BASE_URL=\"http://127.0.0.1:3000\" && npm run serve",
     "start": "bundle exec jekyll serve --drafts --future",
-    "test": "snyk test && npm run axe && npm run crawl && bundle exec rspec && npm run lint",
-    "test:ci": "npm run axe && npm run crawl && bundle exec rspec && npm run lint",
+    "test": "npx snyk test && npm run test:core",
+    "test:detached": "npm run start-detached && npm run test:core",
+    "test:core": "bundle exec rspec && npm run lint && npm run proof && npm run pa11y-ci:sitemap && npm run pa11y-ci:sitemap-mobile",
```

Comment:
Core tests now include both `htmlproofer` and `pa11y`. Run the simpler tests first.